listenbrainz/webserver/views/internet_archive_api.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, request, jsonify
+from listenbrainz.webserver import ts_conn
+from listenbrainz.metadata_cache.internetarchive.store import search_ia_tracks
+
+internet_archive_api_bp = Blueprint('internet_archive_api', __name__)
+
+@internet_archive_api_bp.route('/internet_archive/search', methods=['GET'])
```

Comment:
The prefix should be set when registering the blueprint in init.py, will leave comment there.
```suggestion
@internet_archive_api_bp.get('/search')
```